### PR TITLE
[nodejs] fix wrong nodejs bug versions for skipping tests

### PR DESCRIPTION
**Pull request by rochdev**
## Motivation

The versions that were used focused only on the latest release line and ignored v4.x.

## Changes

Fix wrong Node.js bug versions for skipping tests.

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: tests/appsec/api_security/test_apisec_sampling.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def setup_sampling_rate(self):
     @irrelevant(
         context.library not in ["nodejs"], reason="RFC is deprecated by a newer RFC. New tests will be implemented"
     )
-    @bug(context.library > "nodejs@5.25.0", reason="APPSEC-55901")
+    @bug(context.library > "nodejs@4.49.0", reason="APPSEC-55901")
```
[the remark this answers is not in the document]
Comment (simon-id) [reply tests/appsec/api_security/test_apisec_sampling.py:46]: also need to remove the line just before, but github won't let me

File: tests/test_schemas.py [hunk 2]
```diff
@@ -86,7 +86,7 @@ def test_agent_schema_full(self):
             ]
         )
 
-    @bug(context.library > "nodejs@5.22.0", reason="DEBUG-2864")
+    @bug(context.library > "nodejs@4.46.0", reason="DEBUG-2864")
```
Comment (simon-id) [inline tests/test_schemas.py:89]:
```suggestion
    @bug(context.library > "nodejs@4.46.0", reason="DEBUG-2864") # and 5.22.0
```

File: tests/appsec/api_security/test_apisec_sampling.py [hunk 3]
```diff
@@ -43,7 +43,7 @@ def setup_sampling_rate(self):
     @irrelevant(
         context.library not in ["nodejs"], reason="RFC is deprecated by a newer RFC. New tests will be implemented"
     )
```
Comment (simon-id) [inline tests/appsec/api_security/test_apisec_sampling.py:45]:
```suggestion
```

**Review by simon-id — APPROVED** [review]

**Review by cbeauchesne — CHANGES_REQUESTED** [review]
`./format.sh` is your friend

File: tests/test_schemas.py [hunk 4]
```diff
@@ -86,7 +86,7 @@ def test_agent_schema_full(self):
             ]
         )
 
-    @bug(context.library > "nodejs@5.22.0", reason="DEBUG-2864")
+    @bug(context.library > "nodejs@4.46.0", reason="DEBUG-2864") # and 5.22.0
```
Comment (simon-id) [inline tests/test_schemas.py:89]: lint fix
```suggestion
    @bug(context.library > "nodejs@4.46.0", reason="DEBUG-2864")  # and 5.22.0
```

**Pull request merged**